.buildbot.sh

```diff
@@ -51,45 +52,45 @@ cd sqlite
 mkdir -p build
 cd build
 CC=$GCLANG
-LLVM_COMPILER_PATH=$CUSTOM_CC/bin CC=$CC ../configure
+LLVM_COMPILER_PATH=$LLVM_COMPILER_PATH CC=$CC ../configure
 sed -i '/^CFLAGS =/ s/$/ -fPIC/' Makefile
-make -j32
+LLVM_COMPILER_PATH=$LLVM_COMPILER_PATH make -j32
 # Generate test harness first
-make test -j32
+LLVM_COMPILER_PATH=$LLVM_COMPILER_PATH make test -j32
 # Sqlite3 binaries to split
 binaries=( "sqlite3" "lemon" "mkkeywordhash" "sqlite3_analyzer" 
            "sqltclsh" "fuzzcheck" "sqldiff" "dbhash" "srcck1")
 
 for binary in "${binaries[@]}"; do
    # Extract bitcode from the binary
-   $GET_BC $binary
-   LD_LIBRARY_PATH=$LD_LIBRARY_PATH $SPLITTER $binary.bc -o out-$binary
+   LLVM_COMPILER_PATH=$LLVM_COMPILER_PATH $GET_BC $binary
+   LLVM_EXTRACT=$LLVM_EXTRACT LD_LIBRARY_PATH=$LD_LIBRARY_PATH $SPLITTER $binary.bc -o out-$binary
    cp $repodir/tests/Makefile-sqlite out-$binary/Makefile
    cd out-$binary
-   CC=$CC make
+   LLVM_COMPILER_PATH=$LLVM_COMPILER_PATH CC=$CC make
    cd ..
    mv $binary $binary.original
    # Symlinking to re-use the original test suite
    ln -s $(pwd)/out-$binary/joined $binary
 done
 
 # Reuse the tests suite but with split binaries
-make test
+LLVM_COMPILER_PATH=$LLVM_COMPILER_PATH CC=$CC make test
 
 # Continue with CHERI: cross compile and split lua
 cd $repodir
 
 CHERI=$HOME/cheri/output/sdk
-export LD_LIBRARY_PATH=$CHERI/lib
-export CC=$CHERI/bin/clang 
-export LLVM_EXTRACT=$CHERI/bin/llvm-extract
+LD_LIBRARY_PATH=$CHERI/lib
+CC=$CHERI/bin/clang 
+LLVM_EXTRACT=$CHERI/bin/llvm-extract
 AR=$CHERI/bin/llvm-ar 
 RANLIB=$CHERI/bin/llvm-ranlib
 LLVM_LINK=$CHERI/bin/llvm-link
 LLVM_CONFIG=$CHERI/bin/llvm-config
-export PYTHONPATH=~/build/test-scripts
-export SSH_OPTIONS='-o "StrictHostKeyChecking no"'
-export SCP_OPTIONS='-o "StrictHostKeyChecking no"'
+PYTHONPATH=~/build/test-scripts
+SSH_OPTIONS='-o "StrictHostKeyChecking no"'
+SCP_OPTIONS='-o "StrictHostKeyChecking no"'
```

Comment:
This is actually the ssh options * 2 since `scp -o` takes ssh options :) I'd suggest just having `SSH_OPTIONS`.

Reply:
I *think* this is the same approach we use everywhere but we can modify it here, of course.

Reply:
Simplifications are good, no matter if we haven't done them before.

Reply:
Fix in [cc140f3](https://github.com/capablevms/llvm-function-split/pull/26/commits/cc140f32563318a3fc186cf4bc67e2a890884a4e).